### PR TITLE
Homework5: Ekaterina Zavadskaya

### PR DESCRIPTION
**Pull request by KateZavadskaya**
Add homework5 and minor fix in LICENCE.

### CODE REVIEW

**Review by Stml89 — CHANGES_REQUESTED** [review]
Please fix linters

**Comment by KateZavadskaya** [comment]
@Stml89 It seems I'm successful 💃

File: hm5_1.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+X = LST.index("#")
+LST[X] = "/"
+TXT_12 = "".join(LST)
+
+print(TXT_12)
+
+
+# Способ 3.
+
+print("www.my_site.com#about".replace("#", "/"))
```
Comment (Stml89) [inline hm5_1.py]: it is the same as first solution

File: hm5_3.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+"""
+
+TXT = "Ivanou Ivan"
+
+TXT_2 = TXT.split()
+
+X = TXT_2[0]
+TXT_2[0] = TXT_2[1]
+TXT_2[1] = X
```
Comment (Stml89) [inline hm5_3.py]: not the best solution. 

>>> TXT = "Ivanou Ivan"
>>> TXT_2 = TXT.split()
>>> TXT = " ".join([TXT_2[1], TXT_2[0]])
>>> print(TXT)
Ivan Ivanou

File: hm5_4.py
Comment (Stml89) [inline hm5_4.py:1]: Why do you need a new `any_txt_1` variable?


ANY_TXT = ""
print(f"Исходный текст: '{ANY_TXT}'")

CLEAN_TXT = ANY_TXT.strip()
print(f"Чистый текст: '{CLEAN_TXT}'")

File: hm5_5.py
Comment (Stml89) [inline hm5_5.py:1]: The same, too much extra variables

**Review by Stml89 — COMMENTED** [review]
Please, read my comments and take them into account in the next HWs. Please fix them before merge

**Comment by KateZavadskaya** [comment]
> Please, read my comments and take them into account in the next HWs. Please fix them before merge

Analizing your comment and fixing my mistakes, thank you!👍

**Comment by KateZavadskaya** [comment]
@Stml89 I do my best in resolving your comments. Can you check my results, please.

File: hm5_1.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+
+# Variant 2.
+
+TXT_11 = "www.my_site.com#about"
+
+LST = list(TXT_11)
+X = LST.index("#")
```
Comment (Stml89) [inline hm5_1.py]: why you can not use simple 'find' method?
https://www.w3schools.com/python/ref_string_find.asp

File: hm5_3.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+""" Task 3: В строке “Ivanou Ivan” поменяйте местами слова => "Ivan Ivanou"
+
+"""
+
+TXT = "Ivanou Ivan"
+
+print(TXT.split()[1], TXT.split()[0])
```
Comment (Stml89) [inline hm5_3.py]: here you execute method split() twice, not reaaly good practice 
Better:
something = TXT.split()
print(something[1], something[0])

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hm5_3.py (hunk 4 above)
Comment (KateZavadskaya) [reply hm5_3.py]: I see, will take a note!

File: hm5_1.py (hunk 3 above)
Comment (KateZavadskaya) [reply hm5_1.py]: Understand your message.

File: hm5_3.py (hunk 4 above)
Comment (KateZavadskaya) [reply hm5_3.py:7]: Will redo, ok 👌

Comment (KateZavadskaya) [reply hm5_3.py]: Will redo, ok 👌

File: hm5_1.py (hunk 3 above)
Comment (KateZavadskaya) [reply hm5_1.py]: Will redo, ok 👌

**Comment by KateZavadskaya** [comment]
@Stml89 fixed your comments, check it 🤞, please.

**Review by Stml89 — APPROVED** [review]

**Pull request merged**